packages/backend/src/handlers/graphql/queries/player.ts

```diff
@@ -1,4 +1,18 @@
 export const PlayerQueries = /* GraphQL */ `
+  query GetPlayersByTotalXP {
+    xp(
+      where: {
+        tokenAddress: { _ilike: "0xEAeCC18198a475c921B24b8A6c1C1f0f5F3F7EA0" }
```

Comment:
what is this?

Reply:
It's gonna have to stay hardcoded for now, since we don't have anywhere to fetch other guild token addresses from ATM, and making it dynamic is going to add an unneeded layer of complexity.

What this does, is fetch all Metagame Player RANKS, then recalculate based on the new XP, so it's MG specific and that is SEED token on Polygon network.